WebRoot/templates/calendar/Appointment.template

```diff
@@ -239,6 +239,7 @@
 
 <!-- NOTE: Hopefully the EditView template can go away as soon as task -->
 <!-- NOTE: compose is redesigned to be more like appt compose. -->
+<!-- TODO: Edit View template for Appointment and Task are now separated. Remove conditional code for task-edit.-->
```

Comment:
Linked ticket number in comment: https://github.com/Zimbra/zm-web-client/pull/53/files#diff-60a45c724f7aa5b44add9c6b951de7d2R242